### PR TITLE
Fix/ Moderation - reject related updates

### PR DESCRIPTION
**Pull request by xkopenreview**
this pr should:
1. allow email to be copied to clip board from profile preview modal when clicked (with double quotes)
2. update rejection reason options
3. add a button to insert multiple rejection warning
4. increase rejection message height as the rejection message can be pretty long
5. add user's current institution when rejecting for no institution email

### CODE REVIEW

**Review by emily-grabowski — COMMENTED** [review]
The description makes sense to me

File: pages/user/moderation.js [hunk 1]
```diff
@@ -2207,45 +2207,58 @@ const UserModerationQueue = ({
...
   const rejectionReasons = [
     {
-      value: 'invalidEmail',
-      label: 'Missing Institution Email',
-      rejectionText: `An Institution email matching your latest career/education history is required.\n\n${instructionText}`,
+      value: 'requestEmailVerification',
+      label: 'Institution Email is missing',
+      rejectionText: `Please add and confirm an institutional email ${
+        currentInstitutionName ? `issued by ${currentInstitutionName} ` : ''
+      }to your profile. Please make sure the verficaition token is entered and verified.\n\n${instructionText}`,
```
Comment (racheljsmart) [inline pages/user/moderation.js:2226]: "verficaition" is misspelled, it should be "verification"

File: pages/user/moderation.js [hunk 2]
```diff
@@ -2207,45 +2207,58 @@ const UserModerationQueue = ({
...
+
   const instructionText =
     'Please go back to the sign up page, enter the same name and email, click the Resend Activation button and complete the missing data.'
   const rejectionReasons = [
     {
-      value: 'invalidEmail',
-      label: 'Missing Institution Email',
-      rejectionText: `An Institution email matching your latest career/education history is required.\n\n${instructionText}`,
+      value: 'requestEmailVerification',
+      label: 'Institution Email is missing',
```
Comment (racheljsmart) [inline pages/user/moderation.js:2223]: Change "Institution" to "Institutional"

File: pages/user/moderation.js [hunk 3]
```diff
@@ -2207,45 +2207,58 @@ const UserModerationQueue = ({
...
+      rejectionText: `Please add and confirm an institutional email ${
+        currentInstitutionName ? `issued by ${currentInstitutionName} ` : ''
+      }to your profile. Please make sure the verficaition token is entered and verified.\n\n${instructionText}`,
     },
     {
-      value: 'imPersonalHomepage',
-      label: 'Impersonal Homepage',
-      rejectionText: `The homepage url provided in your profile is invalid or does not display your name so your identity can't be determined.\n\n${instructionText}`,
+      value: 'requestEmailConfirmation',
+      label: 'Institution Email is added but not confirmed',
```
Comment (racheljsmart) [inline pages/user/moderation.js:2230]: Change "Institution" to "Institutional"

File: pages/user/moderation.js [hunk 4]
```diff
@@ -2207,45 +2207,58 @@ const UserModerationQueue = ({
...
+      rejectionText: `Please confirm the institutional email in your profile by clicking "Confirm" button next to the email and enter the verification token received.\n\n${instructionText}`,
```
Comment (racheljsmart) [inline pages/user/moderation.js:2231]: change to 'clicking the "Confirm" button'

File: pages/user/moderation.js [hunk 5]
```diff
@@ -2207,45 +2207,58 @@ const UserModerationQueue = ({
...
     },
     {
-      value: 'imPersonalHomepageAndEmail',
-      label: 'Impersonal Homepage + Missing Institution Email',
-      rejectionText: `A Homepage url which displays your name and institutional email matching your latest career/education history are required.\n\n${instructionText}`,
+      value: 'invalidDBLP',
+      label: 'DBLP is disambiguation page',
```
Comment (racheljsmart) [inline pages/user/moderation.js:2235]: change to "DBLP link is a disambiguation page"

File: pages/user/moderation.js [hunk 6]
```diff
@@ -2207,45 +2207,58 @@ const UserModerationQueue = ({
...
+      label: 'Homepage is invalid',
+      rejectionText: `The homepage url provided in your profile is invalid or does not display your name so your identity can't be determined.\n\n${instructionText}`,
     },
     {
-      value: 'requestEmailVerification',
-      label: 'Request Email Verification',
-      rejectionText: `Please add and confirm an institutional email to your profile.\n\n${instructionText}`,
+      value: 'imPersonalHomepageAndEmail',
+      label: 'Homepage is invalid + no institution email',
+      rejectionText: `A Homepage url which displays your name and institutional email matching your latest career/education history are required. Please confirm the institution email by entering the verification token received after clicking confirm button next to the institutional email.\n\n${instructionText}`,
```
Comment (racheljsmart) [inline pages/user/moderation.js:2246]: change "institution email" to "institutional email"

File: pages/user/moderation.js [hunk 7]
```diff
@@ -2207,45 +2207,58 @@ const UserModerationQueue = ({
...
+      value: 'lastNotice',
+      label: 'Last notice before block',
+      rejectionText: `If invalid info is submitted again, your email will be blocked.\n\n${instructionText}`,
     },
   ]
 
+  const updateMessageForPastRejectProfile = () => {
+    setRejectionMessage(
+      (p) =>
+        `Submitting invalid info is a violation of OpenReview's Terms and Conditions which may result in terminating your access to the system.\n\n${p}`
```
Comment (racheljsmart) [inline pages/user/moderation.js:2258]: Can we hyperlink OpenReview's Terms and Conditions so users can check out the page?

File: pages/user/moderation.js [hunk 8]
```diff
@@ -2207,45 +2207,58 @@ const UserModerationQueue = ({
...
+const RejectionModal = ({ id, profileToReject, rejectUser, signedNotes }) => {
   const [rejectionMessage, setRejectionMessage] = useState('')
   const selectRef = useRef(null)
 
+  const currentInstitutionName = profileToReject?.content?.history?.find(
+    (p) => !p.end || p.end >= new Date().getFullYear()
+  )?.institution?.name
+
   const instructionText =
     'Please go back to the sign up page, enter the same name and email, click the Resend Activation button and complete the missing data.'
```
Comment (emily-grabowski) [inline pages/user/moderation.js:2219]: "click the Resend Activation button and follow the activation link to update your information'

**Review by emily-grabowski — COMMENTED** [review]
A couple of comments on the text and wordign

**Review by racheljsmart — APPROVED** [review]

**Review by emily-grabowski — APPROVED** [review]

**Pull request merged**